### DISCUSSION TITLE
utf8_decode on php 8.2

### DISCUSSION THREAD

**Discussion by Max84** [PHP-CS-Fixer/PHP-CS-Fixer#6852] [Q&A]
## Support question

Well, I'm not sure it's a bug, so I prefer to ask it as support for now.
I saw #6609 .

I understand rule @PHP82Migration should replace deprecated utf8_decode.
But, CS Fixer doesn't replace utf8_decode into my php script and doesn't alert me too.

Did I miss something?

Thanks

**Comment by kubawerlos** [comment]
> I understand rule @PHP82Migration should replace deprecated utf8_decode.

How did you come to this conclusion?

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6609 is only about removing those deprecated functions in `mb_convert_encoding`.

In [RFC](https://wiki.php.net/rfc/remove_utf8_decode_and_utf8_encode) there are some hints about what the replacement could/should be, but there is no straight-forward strategy that will always work, thus I don't see a possibility for a fixer there.

**Comment by Max84** [comment]
Thanks @kubawerlos 
I might misunderstand #6609 !
So, can CS Fixer notify us if some php code can't be fixed?
These notifications already exist for some other rules.

**Reply by Wirone** [reply]
No, Fixer does not report problems it can't fix, that's the basic requirement 🙂.
